### DISCUSSION TITLE
Preciso de ajuda com um processo de lembrete/cobrança automatica usando Python

### DISCUSSION THREAD

**Discussion by mcgbns** [frontendbr/forum#2456] [Discussão]
Basicamente, o processo todo ja está feito, porém por algum motivo desconhecido, a minha mensagem acaba não sendo enviada, e não parte para o próximo contato...

quem puder me dar um apoio, sou iniciante na area porém com um interesse imenso em aprender com a forma prática,

**Comment by igorlvicente** [comment]
Você tem que dar mais detalhes do que você está falando.
